### PR TITLE
Miscellaneous clean up

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Clean up some things I saw while observing [this action](https://github.com/NYCPlanning/db-pluto/actions/runs/2664592879)
These changes shouldn't effect functionality, just make actions easier to understand and reduce non-helpful things reported to logs.

### CODE REVIEW

File: pluto_build/bin/config.sh [hunk 1]
```diff
@@ -146,10 +146,8 @@ function import_qaqc {
     echo "✅ $name.sql exists in cache"
   else
     echo "🛠 $name.sql doesn't exists in cache, downloading ..."
-    echo "{$url}/{$name.sql}"
```
Comment (td928) [inline pluto_build/bin/config.sh:149]: I do find this print out sometimes helpful with getting a visual on url being curled e.g. when the url is wrong or the version of the source is wrong with debugging purposes in mind.

Comment (td928) [reply pluto_build/bin/config.sh:149]: I do think the url itself is useful as well but open to others' opinions on this if it seems excessive to everyone else.

Comment (SashaWeinstein) [reply pluto_build/bin/config.sh:149]: I doubt anyone else will have an opinion on this lol, I'll add back that echo statement lol

**Review by mbh329 — APPROVED** [review]

File: pluto_build/bin/config.sh (hunk 1 above)
Comment (SashaWeinstein) [reply pluto_build/bin/config.sh:149]: Ok Max is on my side I'm taking it back out

**Review by td928 — APPROVED** [review]

**Pull request merged**